src/_pytest/assertion/rewrite.py

```diff
@@ -44,6 +44,10 @@
 if TYPE_CHECKING:
     from _pytest.assertion import AssertionState
 
+if sys.version_info >= (3, 11):
+    from importlib.resources.readers import FileReader
+elif sys.version_info >= (3, 10):
+    from importlib.readers import FileReader
```

Comment:
this change isn't needed -- and should remain inline in the function since it's verion-specific